examples/transforms/global_vs_local_translation.rs

```diff
@@ -0,0 +1,141 @@
+use bevy::prelude::*;
+
+// Define a struct to store basic state information.
+struct EntityState {
```

Comment:
I'm don't think this is the clearest way to store this information: generally, I'd pass in the spawn location to the entity spawning system, and then have a separate `Stopped` or `IsStopped(bool)` component.

Reply:
I'd change it to Moving (instead of Stopped) and remove the component once it should stop. Also makes filtering easier since it's in the queries and is probably more "the bevy/ecs way"?
I assume that was on your mind?